src/at2/Man.java

```diff
@@ -0,0 +1,31 @@
+package at2;
+
+public class Man extends Human {
+    public final String sex = "male";
+    private boolean beard;
+
+    Man() {};
+
+    Man(String name, int age, double height, double weight, String hairColor) {
+        super(name, age, height, weight, hairColor);
+    }
+
+    public String getSex() {
+        return "male";
+    }
+
+    public void setBeard(boolean beard) {
+        beard = getAge() > 16 ? this.beard = beard : false;

```

Comment:
let's think on what are you doing here:
1. pass `boolean beard` as method input parameter
2. try to re-assign `beard` variable (that one that is acting like a method input parameter, and, even that is your method idea, this line could be simplified to `beard = getAge() > 16 && (this.beard = beard);`)
3. what about object inner state (meaning `this.beard` variable) assignment?